integration/token/fungible/tests.go

```diff
@@ -209,6 +209,9 @@ var BobAcceptedTransactions = []*ttxdb.TransactionRecord{
 func TestAll(network *integration.Infrastructure, auditor string) {
 	RegisterAuditor(network, auditor)
 
+	// give some time to the nodes to get the public parameters
+	time.Sleep(10 * time.Second)
```

Comment:
Is there a way to move this sleep somewhere so it doesn't have to appear at the beginning of each test?